### PR TITLE
feat: add filter on worst buses

### PR DESCRIPTION
**Pull request by moshe742**

### CODE REVIEW

File: src/pages/dashboard/DashboardPage.tsx [hunk 1]
```diff
@@ -28,6 +31,10 @@ const DashboardPage = () => {
   const [endDate, setEndDate] = useDate(now.clone().subtract(1, 'day'))
   const [groupByHour, setGroupByHour] = React.useState<boolean>(false)
 
+  const { search, setSearch } = useContext(SearchContext)
+  const { operatorId, lineNumber, timestamp, routes, routeKey } = search
```
Comment (NoamGaash) [inline src/pages/dashboard/DashboardPage.tsx:35]:
```suggestion
  const [operatorId, setOperatorId] = useState('')
```
will be easier.
we don't need the entire search context here

**Review by NoamGaash — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Please consider my comment (I'm available to discuss it).

quick tip - you can run the linter locally using the `npm run lint` command

File: src/pages/dashboard/DashboardPage.tsx (hunk 1 above)
Comment (moshe742) [reply src/pages/dashboard/DashboardPage.tsx]: Done

**Review by NoamGaash — APPROVED** [review]
great! thank you :pray:

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**